scripts/cnode-helper-scripts/guild-deploy.sh

```diff
@@ -110,6 +110,7 @@ set_defaults() {
   [[ "${SUDO}" = 'Y' ]] && sudo="sudo" || sudo=""
   [[ "${SUDO}" = 'Y' && $(id -u) -eq 0 ]] && err_exit "Please run as non-root user."
   [[ -z "${CARDANO_NODE_VERSION}" ]] && CARDANO_NODE_VERSION="$(curl -sfk "https://raw.githubusercontent.com/${G_ACCOUNT}/guild-operators/${BRANCH}/files/docker/node/release-versions/cardano-node-latest.txt")"
+  [[ -z ${MITHRIL_HOME} ]] && MITHRIL_HOME="${CNODE_HOME}/mithril"
```

Comment:
sorry - this needs to be a line below (next line defines `CNODE_HOME`)

Reply:
Woops. Test suite running on an already deployed VM with guild ops. The MITHRIL_HOME just be set from the environment already as the directory was created. Fixing now.